internal/monitors/statsd/metadata.yaml

```diff
@@ -0,0 +1,75 @@
+- dimensions: null
+  doc: |
+
+
+    ### USAGE
+
+    #### Adding dimensions to StatsD metrics
+
+    The StatsD monitor can parse keywords from a statsd metric name by a set of
+    converter that was configured by user.
+
+    ```
+    converters:
+      - pattern: "cluster.cds_{traffic}_{mesh}_{service}-vn_{protocol}_{port}.{action}"
+        dimensions:
+          - traffic
+          - mesh
+          - service
+          - action
+    ```
+
+    This converter will parse `traffic`, `mesh`, `service` and `action` as dimensions
+    from a metric name `cluster.cds_egress_ecommerce-demo-mesh_gateway-vn_tcp_8080.update_success`.
+
+    Metrics will be sent without dimensions if there's no related configuration.
+
+    #### Formatting metric name
+
+    You can customize a metric name by providing a format string within the converter configuration.
+    
+    ```
+    converters:
+      - pattern: "cluster.cds_{traffic}_{mesh}_{service}-vn_{protocol}_{port}.{action}"
+        metric: "{traffic}.{action}"
+    ```
+    
+    The metrics which match to the given pattern will be reported to SignalFx as `{traffic}.{action}`.
```

Comment:
Are `traffic` and `action` also dimensions in this case or not?

Reply:
`dimensions` config wasn't given, so they won't be any dimensions in this case.

Reply:
I think if they named a section in the pattern, it should be a dimension automatically.  If you want to be able to "throw away" sections without adding them as dimensions, just let them do something like `{*}` or something, but I think in most cases they will want the diemnsions anyway.

Reply:
I'll allow `{}` for non-collecting section. Concerned if `*` give the user a wrong impression that we support wildcards.